tests/src/test/scala/whisk/core/controller/test/ActivationsApiTests.scala

```diff
@@ -462,6 +462,57 @@ class ActivationsApiTests extends ControllerTestCommon with WhiskActivationsApi
     }
   }
 
+  it should "skip activations and return correct ones" in {
+    implicit val tid = transid()
```

Comment:
@chetanmeh do you mind taking a look at these skip related tests as it’s something you’ve tweaked in the past.